content/aips/AIP-35.md

```diff
@@ -0,0 +1,120 @@
+---
+aip: 35
+title: Adjust interest rate to account for APY over-approximation
+status: Proposed
+author: Ahmed Naguib Aly (@ahnaguib)
+shortDescription: Adjust interest rate to account for APY over-approximation
+discussions: https://governance.aave.com/t/arc-update-ampl-interest-rate-curve-to-account-for-over-approximation-in-compounded-interest/5350
+created: 2021-07-26
+---
+
+## Simple Summary
+
+- Update interest rate on AMPL market to account for over-approximation in compounded interest computation.
+- Make use of the exponential curve due to the over-approximation to set higher max APY.
+
+
+## Motivation
+
+In AAVEs MathUtils an over-approximation in the application of the interest rate for computing compounded interest was discovered. 
```

Comment:
Lines 19 and 21 seem to duplicate each other. Only need one of them?

Reply:
Good catch. Fixed.